### PR TITLE
Add doc for http solver ingressTemplate

### PR DESCRIPTION
**Pull request by zeeZ**
jetstack/cert-manager/pull/2023

I've added some common examples. Anywhere else to update?

### CODE REVIEW

**Comment by meyskens** [comment]
@munnerz I changed the branch to merge this into to next, however it seems it already got very out of date. We should set something up to fix this.

File: content/en/docs/configuration/acme/http01/_index.md [hunk 1]
```diff
@@ -121,3 +121,36 @@ The added labels and annotations will merge on top of the cert-manager defaults,
...
 
 No other fields of the `podTemplate` exist.
+
+### `ingressTemplate`
+
+It is possible to add labels and annotations to the solver ingress resources.
+These can be configured under the `metadata` field under `ingressTemplate`:
+
+```yaml
+apiVersion: cert-manager.io/v1alpha2
```
Comment (meyskens) [inline content/en/docs/configuration/acme/http01/_index.md]: Can we make the version `v1alpha3`?

Comment (munnerz) [reply content/en/docs/configuration/acme/http01/_index.md]: All docs samples are currently v1alpha2, and given a subset of users will be unable to use v1alpha3 yet, I think it makes sense for us to hold these back for now (and update all manifests at once). Consistency is best IMO 🙂

Comment (meyskens) [reply content/en/docs/configuration/acme/http01/_index.md]: :+1:

**Comment by munnerz** [comment]
@zeeZ you'll need to update your PR so it is made against the `release-next` branch, as this change will only become available in the *next* release, hence we have a separate branch to track these changes :)

Can you `git rebase -i release-next` to fix up these extra commits, and then `git push -f origin add-ingress-template-doc` to force push the changes to this PR?

Thanks!

**Comment by zeeZ** [comment]
@munnerz Done!

**Pull request merged**